src/assets/locales/resources_en_AU.json

```diff
@@ -0,0 +1,14 @@
+{
+  "skyux_date_range_picker_format_label_last_fiscal_year": {
+    "_description": "The label for the date range picker component's select field Last Financial Year format type.",
```

Comment:
Only the `en_US` resource file should have a `_description` property as it's only used to provide context to translators. The strings in these files should only contain the `message` property.